pkg/tools/iacinventory/walk.go

```diff
@@ -14,7 +14,7 @@ func fileTooLarge(info os.FileInfo) bool {
 	return info.Size() > maxSize
 }
 
-func isCloudFormationFile(path string, info os.FileInfo) bool {
+func IsCloudFormationFile(path string, info os.FileInfo) bool {
```

Comment:
Funcs exported so that I can perform this walk again in the `iac-scan` subtools while only sending the directory to the API in `iacinventory`.